### ISSUE TITLE
Transpile to C#

### ISSUE THREAD

**Issue by munrocket** [xpl/crosspile#1]
Can you add transpile to C#? Your library looks pretty good

**Comment by munrocket** [comment]
Here you can find tranpile project https://github.com/alehander42/pseudo-python/

**Comment by xpl** [comment]
Yeah, we plan to add support for many more languages, as this is the ultimate goal of the project!

**Comment by munrocket** [comment]
I finded different package for goal, perhaps this will help  you
1) Haxe https://github.com/HaxeFoundation/haxe
2) Pseudo https://github.com/alehander42/pseudo-python/

**Comment by Saad-Alothman** [comment]
This is a great library , although i wish for me personally it was in C#
